frontend/packages/topology/src/filters/TopologyFilterBar.scss

```
@@ -1,12 +1,14 @@
 @import '../../../../public/style/vars';
 
+
+
 .odc-topology-filter-bar {
   background-color: var(--pf-global--BackgroundColor--100);
-  padding: 6px $pf-global-gutter;
+  padding: 6px 0rem;
```

Not tested it yet, but when you want to say zero, you don't need a unit.

```suggestion
  padding: 6px 0;
```